mmv1/third_party/terraform/tests/resource_network_security_client_tls_policy_test.go.erb

```diff
@@ -0,0 +1,69 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccNetworkSecurityClientTlsPolicy_update(t *testing.T) {
```

Comment:
Can you please add the updating tests for other fields, too? Thanks.

Reply:
Added